dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-wechat/src/main/java/org/apache/dolphinscheduler/plugin/alert/wechat/WechatAppMessage.java

```diff
@@ -29,8 +29,8 @@ public class WechatAppMessage {
     private Map<String, String> text;
     private Map<String, String> markdown;
     private Integer safe;
-    private Integer enable_id_trans;
-    private Integer enable_duplicate_check;
+    private Integer enableIdTrans;
```

Comment:
I'm not sure it is ok, please check it.

Reply:
https://github.com/apache/dolphinscheduler/blob/2397423eb6893458cd13866dd16b2c2e92dda437/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-wechat/src/main/java/org/apache/dolphinscheduler/plugin/alert/wechat/WeChatSender.java#L217-L218

I think this may cause wechat send message error. We should avoid serializing these parameters.

Reply:
It should be the specified field of the document, this modification is restored in new pr 10922. #10922

Reply:
Please revert these filedNames in this PR, and if someone cherry-pick this PR, this will import a new BUG.